test/jake/jakelib/migrate-test.js

```diff
@@ -151,6 +151,123 @@ test('migrate:rollback prints verbose logs', (temp) => {
   });
 });
 
+test('migrate:rollback --all rolls back all completed migrations', (temp) => {
+  return (
+    new Promise((resolve, reject) => {
+      fs.writeFile(
```

Comment:
I would recommend using writeFileSync for simplicity, we don't care about performance in tests much.